source/faqs.rst

```diff
@@ -373,3 +367,208 @@ The ``--no-builds`` argument completely removes the build number from the
+package is originally *built* differs from the version when the package is
+*installed*. All packages intending to be installed into the same environment
+should be built using the same versions of common libraries so that they can
+co-exist. **Global pinning** is the idea of making sure all recipes use the
+same versions of common libraries.
+
+For example, many bioinformatics tools have ``zlib`` as a dependency.
+The version of ``zlib`` used when building the package should be the same as the
+version used when installing the package into a new environment. This implies
+that we need to specify the ``zlib`` version in one place and have *all recipes
+intended to coexist in the same environment* use that version.
+
+This is configured with special build config files. Since we rely heavily on
+the conda-forge channel, the bioconda build system installs the conda-forge
+`conda_build_config.yaml
+<https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml>`_
+into our build environment so that it can be used for building all recipes.
+This is then combined with the bioconda-specific
+`bioconda-Utils-conda_build_config.yaml
+<https://github.com/bioconda/bioconda-utils/blob/master/bioconda_utils/bioconda_utils-conda_build_config.yaml>`_.
+Note that in some cases the bioconda config may override some of the
+conda-forge configs. For example, historically, we did this when we wanted to
+support older Python versions.
+
+The idea here is to specify that any time a dependency (``zlib``, in our
+running example) is used as a build dependency, it should also be automatically
+be installed as a run dependency without having to explicitly add it as such in
+the recipe. This specification is done in the ``zlib`` recipe itself (which is
+hosted by conda-forge), so in general bioconda collaborators can just add
+``zlib`` as a build dependency.
+
+Note that we don't have to specify the version of ``zlib`` in the recipe -- it
+is pinned in that ``conda_build_config.yaml`` file we share with conda-forge.
+
+In a similar fashion, the reason that we don't have to specify ``libgcc`` as
+a *run* dependency is that ``{{ compiler('c') }}`` automatically exports
+``libgcc`` as a run dependency of any package that uses the C compiler to
+build.
+
+.. _platform-nomenclature-faq:
+
+Understanding platform nomenclature
+-----------------------------------
+
+.. datechanged:: 2023-11-18
+   Added section
+
+Different CPU chips use different architecture, so programs are written
+fundamentally differently for them. A package with compiled dependencies must
+therefore use platform-specific dependencies.
+
+There is a lot of confusing nomenclature surrounding them. Here is an attempt
+at clearing them up, or at least providing enough context that you can look up
+more details on your own:
+
+**instruction set, CISC, RISC, RISC-V**: The *instruction set* is the assembly
```

Comment:
I think `ISA` (instruction set architecture) is the most common term here.

Reply:
done